docs/dev/new_check_howto.md

```diff
@@ -339,7 +350,9 @@ ddev validate config --sync awesome
 {{% /tab %}}
 {{% tab "Manifest" %}}
 
-The `awesome/manifest.json` for the Awesome Service Check. **Note**: The `guid` must be unique (and valid), so do _not_ use the one from this example (the tooling generates one for you):
+The `awesome/manifest.json` for the Awesome Service Check:
```

Comment:
Let's include a link to the manifest.json documentation: https://docs.datadoghq.com/developers/integrations/check_references/#manifest-file

Reply:
We should also update the below manifest.json example to use a manifest v2 example

Reply:
Good call! I'll work on adding these

Reply:
@fanny-jiang Should we also add a line that shows users how to migrate their manifest to the newer version? (`ddev meta manifest migrate awesome 2.0.0`) When I used the tools to generate an integration, it defaulted to the V1 manifest.

Reply:
I believe manifest version 2 is the default for creating new integrations. Do you have an older version of ddev installed by any chance?

Reply:
Since this guide is mainly for users creating a new integration, I don't think we need a line showing how to migrate to the newer manifest version.

Reply:
Great! I agree we don't need to add it if it's included by default in the newer versions.